src/registrar/admin.py

```diff
@@ -2864,34 +2863,204 @@ def save_model(self, request, obj, form, change):
+    fieldsets = [
+        # created_on is the created_at field, and portfolio_type is f"{organization_type} - {federal_type}"
+        (None, {"fields": ["portfolio_type", "organization_name", "creator", "created_on", "notes"]}),
+        # TODO - uncomment in #2521
+        # ("Portfolio members", {
+        #     "classes": ("collapse", "closed"),
+        #     "fields": ["administrators", "members"]}
+        # ),
+        ("Portfolio domains", {"classes": ("collapse", "closed"), "fields": ["domains", "domain_requests"]}),
+        ("Type of organization", {"fields": ["organization_type", "federal_type"]}),
+        (
+            "Organization name and mailing address",
+            {
+                "fields": [
+                    "federal_agency",
+                    "state_territory",
+                    "address_line1",
+                    "address_line2",
+                    "city",
+                    "zipcode",
+                    "urbanization",
+                ]
+            },
+        ),
+        ("Suborganizations", {"fields": ["suborganizations"]}),
+        ("Senior official", {"fields": ["senior_official"]}),
+    ]
+
+    # This is the fieldset display when adding a new model
+    add_fieldsets = [
+        (None, {"fields": ["organization_name", "creator", "notes"]}),
+        ("Type of organization", {"fields": ["organization_type"]}),
+        (
+            "Organization name and mailing address",
+            {
+                "fields": [
+                    "federal_agency",
+                    "state_territory",
+                    "address_line1",
+                    "address_line2",
+                    "city",
+                    "zipcode",
+                    "urbanization",
+                ]
+            },
+        ),
+        ("Senior official", {"fields": ["senior_official"]}),
+    ]
 
     list_display = ("organization_name", "federal_agency", "creator")
     search_fields = ["organization_name"]
     search_help_text = "Search by organization name."
     readonly_fields = [
-        "creator",
+        # This is the created_at field
+        "created_on",
+        # Custom fields such as these must be defined as readonly.
+        "federal_type",
+        "domains",
+        "domain_requests",
+        "suborganizations",
+        "portfolio_type",
     ]
 
+    def federal_type(self, obj: models.Portfolio):
+        """Returns the federal_type field"""
+        return BranchChoices.get_branch_label(obj.federal_type) if obj.federal_type else "-"
+
+    federal_type.short_description = "Federal type"  # type: ignore
+
+    def created_on(self, obj: models.Portfolio):
+        """Returns the created_at field, with a different short description"""
+        # Format: Dec 12, 2024
+        return obj.created_at.strftime("%b %d, %Y") if obj.created_at else "-"
+
+    created_on.short_description = "Created on"  # type: ignore
+
+    def portfolio_type(self, obj: models.Portfolio):
```

Comment:
Blocking question: I don't understand the difference / inter-relationships between portfolio type, organization type, federal type, federal agency